code/modules/martial_arts/combos/sleeping_carp/keelhaul.dm

```diff
@@ -6,13 +6,15 @@
 /datum/martial_combo/sleeping_carp/keelhaul/perform_combo(mob/living/carbon/human/user, mob/living/target, datum/martial_art/MA)
 	user.do_attack_animation(target, ATTACK_EFFECT_KICK)
 	playsound(get_turf(target), 'sound/effects/hit_kick.ogg', 50, TRUE, -1)
-	if(!target.IsWeakened() && !IS_HORIZONTAL(target) && !target.stat)
+	if(!IS_HORIZONTAL(target))
 		target.apply_damage(10, BRUTE, BODY_ZONE_HEAD)
-		target.Weaken(4 SECONDS)
+		target.KnockDown(4 SECONDS)
```

Comment:
```suggestion
		target.KnockDown(6 SECONDS)
```
considering carp isn't the strongest would you increase this number slightly?

Reply:
Yeah, that's a good shout. I'll get on it.